pkg/controllers/validatingadmissionpolicy-generate/controller.go

```diff
@@ -326,34 +326,50 @@ func (c *controller) reconcile(ctx context.Context, logger logr.Logger, key, nam
 		return nil
 	}
 
+	vapName := policy.GetName()
+	vapBindingName := vapName + "-binding"
```

Comment:
Better to have an util construct the name.

Reply:
Done.